packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -936,95 +932,45 @@ class ServiceExtensionManager {
   }
 
   ValueListenable<bool> hasServiceExtensionListener(String name) {
```

Comment:
can we rename to hasServiceExtensionListenable?

Reply:
done. renamed to hasServiceExtension